pubspec.yaml

```diff
@@ -56,7 +56,8 @@ flutter:
   assets:
     - assets/
     - assets/items/
-    - images/
+    - assets/items/google.png
```

Comment:
No changes in this file is required, line 58 will do the work

Reply:
Ok